src/lib/ConcreteSyntaxData.ml

```diff
@@ -55,7 +55,6 @@ and con_ =
   | BotC
   | HCom of con * con * con * con * con
   | HFill of con * con * con * con
-  | AutoHCom of con * con * con * con
```

Comment:
I thought you still want to keep this?

Reply:
The fact that I deleted it means that I am getting rid of it for now (there is some subtlety with El that I want to work out)